tests/unit/test_platforms_get_platform.py

```diff
@@ -203,3 +206,7 @@ def test_get_platform_warn_mode_fail_if_backticks():
         r'platform = `echo \$\{chamber\}`: '
         r'backticks are not supported; please use \$\(\)'
     )
+
+
+def test_get_localhost_install_target():
+    assert get_localhost_install_target() == 'localhost'
```

Comment:
```suggestion
    assert get_localhost_install_target() == 'localhost'

```

Reply:
Ah, I think this is to do with the way the diff is displayed on github. Blank line 213 was there, just not displayed as it was no change to the original?